OpenRA.Mods.Cnc/Traits/PortableChrono.cs

```diff
@@ -88,9 +88,9 @@ public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool qu
 				self.World.OrderGenerator = new PortableChronoOrderGenerator(self, Info);
 
 			if (order.OrderID == "PortableChronoTeleport")
-				return new Order(order.OrderID, self, queued) { TargetLocation = self.World.Map.CellContaining(target.CenterPosition) };
+				return new Order(order.OrderID, self, target, queued);
 
-			return new Order(order.OrderID, self, queued) { TargetActor = target.Actor };
+			return null;
```

Comment:
Before here was also created Order for "PortableChronoDeploy", now it is null - intentional, right?

Reply:
Yeah, this was a bug that would duplicate other order types.